users/tests/test_user_signup.py

```diff
@@ -42,3 +44,11 @@ def test_sign_up_email_already_exists(self):
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data['email'], [SignUpTests.ALREADY_REGISTERED])
+
+    def test_confirmation_email_sent(self):
+        self.data.update({'password1': self.password, 'password2': self.password})
+        self.client.post('/dj-rest-auth/registration/', self.data)
```

Comment:
You can use [reverse](https://docs.djangoproject.com/en/3.1/ref/urlresolvers/) here, in this way if the URL of the endpoint change you don't have to change in all the places it has been used

Reply:
Yes, just as `self.sign_in_url`